kube_helpers.go

```diff
@@ -127,7 +73,7 @@ func annotateNamespace(ns string, labels map[string]string) {
 
 		exitCode, _, _ := cmd.exec(debug, verbose)
 		if exitCode != 0 && verbose {
-			log.Println("WARN: I could not annotate namespace [ " + ns + " with " + k + "=" + v +
+			logs.Info("I could not annotate namespace [ " + ns + " with " + k + "=" + v +
```

Comment:
I would remove the "personnification" in the logs here to be consistent with log message refactoring. Maybe `namespace cannot be annotated [...]. It already exists. Skipping..`

Reply:
:+1: